Safari/Info.plist

```diff
@@ -13,9 +13,9 @@
 	<key>CFBundleInfoDictionaryVersion</key>
 	<string>6.0</string>
 	<key>CFBundleShortVersionString</key>
-	<string>7.7.3</string>
+	<string>7.7.4</string>
 	<key>CFBundleVersion</key>
-	<string>1</string>
+	<string>7.7.4</string>
```

Comment:
@nightpool You were right! Work ate more of my brain then I thought it would but the basic gist is that bundleversion is just a thing that can be changed in the Extension Builder UI.

In addition to updating this version, could you update the value of Builder Version to <string>12602.4.8</string>? It corresponds to the newer version of Safari I'll be using to package the extension.